beacon-chain/rpc/eth/beacon/blocks.go

```diff
@@ -236,21 +248,21 @@ func (bs *Server) SubmitBlock(ctx context.Context, req *ethpbv2.SignedBeaconBloc
 			return nil, status.Errorf(codes.InvalidArgument, "Could not tree hash block: %v", err)
 		}
 
+		// Broadcast the new block to the network.
+		if err := bs.Broadcaster.Broadcast(ctx, wrappedBellatrixBlk.Proto()); err != nil {
+			return nil, status.Errorf(codes.Internal, "Could not broadcast block: %v", err)
+		}
+
 		// Do not block proposal critical path with debug logging or block feed updates.
 		defer func() {
 			log.WithField("blockRoot", fmt.Sprintf("%#x", bytesutil.Trunc(root[:]))).Debugf(
 				"Block proposal received via RPC")
 			bs.BlockNotifier.BlockFeed().Send(&feed.Event{
 				Type: blockfeed.ReceivedBlock,
-				Data: &blockfeed.ReceivedBlockData{SignedBlock: wrappedBellatrixBlk},
+				Data: &blockfeed.ReceivedBlockData{SignedBlock: wrappedBellatrixBlk, IsOptimistic: false},
```

Comment:
This one is bellatrix block, have we decided to just return `false` anyway?
Because we don't care about this endpoint?

Reply:
@potuz mentioned that a proposed block is always optimistic. My thinking is that in this case we should always return `true` here. I must have copy/pasted `false` by mistake.

Reply:
Is it? Why would a validator propose an optimistic block? That's a violation of the spec

cc @potuz